hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -240,15 +240,14 @@ static int xpadneo_ff_play(struct input_dev *dev, void *data,
 	 * so we can completely turn off the main rumble while still doing
 	 * trigger rumble alone
 	 */
-	if (direction <= DIRECTION_UP) {
+	if (direction <= DIRECTION_UP)
 		/* scale the main rumbling between 0x0000..0x8000 (100%..0%) */
 		fraction_MAIN =
 		    ((DIRECTION_UP - direction) * 100) / DIRECTION_UP;
-	} else {
+	else
```

Comment:
I don't think we should omit braces when a comment is involved. It enables to pitfalls too easily.

Reply:
What pitfalls, precisely?

Reply:
The indented part already splits across multiple lines. One may be tempted to actually split the statement within the indention without the compiler noticing any problems, and neither the human (since it looks fine indention-wise).

Reply:
So should a comment count towards the number of statements in an `if` then? I can put the braces back in here.

Reply:
The braces here are back in.

Reply:
> So should a comment count towards the number of statements in an `if` then? I can put the braces back in here.

Yes, I think that's better for authors not yet familiar with the code. Especially long comments may confuse people about the curly braces.